### PR TITLE
Change prerelease versioning

### PR DESCRIPTION
**Pull request by weshaggard**
- Use a instead of dev

Updating versioning logic to match guideline update in Azure/azure-sdk#1536.

PTAL @lmazuel @praveenkuttappan @scbedd

### CODE REVIEW

**Review by scbedd — DISMISSED** [review]
LGTM

File: eng/versioning/version_set_dev.py [hunk 1]
```diff
@@ -25,8 +25,7 @@ def format_build_id(build_id):
 
 def get_dev_version(current_version, build_id):
     parsed_version = parse(current_version)
-    #release = parsed_version.release
-    return "{0}.dev{1}".format(parsed_version, build_id)
+    return "{0}a{1}".format(parsed_version.base_version, build_id)
```
Comment (scbedd) [inline eng/versioning/version_set_dev.py:28]: From this:
image

to this:
image

Comment (weshaggard) [reply eng/versioning/version_set_dev.py:28]: This is what I intended to do. Is there something I need to fix because of that change?

**Pull request merged**